cmd/config-host-list.go

```diff
@@ -94,7 +94,12 @@ func printHosts(hosts ...hostMessage) {
 	}
 	for _, host := range hosts {
 		// Format properly for alignment based on alias length.
-		host.Alias = fmt.Sprintf("%-*.*s", maxAlias, maxAlias, host.Alias)
+		host.Alias = fmt.Sprintf("%-*.*s :", maxAlias, maxAlias, host.Alias)
+		if host.AccessKey == "" || host.SecretKey == "" {
+			host.AccessKey = ""
+			host.SecretKey = ""
+			host.API = ""
```

Comment:
> Perhaps we can save empty API string when the config host add was called first time? instead of doing this later?

I think it is better if we do that here since users can modify config.json manually.. we can't prevent them.